exec/java-exec/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -297,4 +297,44 @@ SqlNode SqlDescribeSchema() :
    {
         return new SqlDescribeSchema(pos, schema);
    }
+}
```

Comment:
Looks fine conceptually, but have someone double-check this who is more familiar with the details of the parser implementation.